include/jwt-cpp/jwt.h

```diff
@@ -967,12 +967,28 @@ namespace jwt {
 				size_t len = EVP_PKEY_size(pkey.get());
 				std::string res(len, '\0');
 
+				// LibreSSL is the special kid in the block, as it does not support EVP_DigestSign.
+				// OpenSSL on the otherhand does not support using EVP_DigestSignUpdate for eddsa, which is why we end up with this mess.
```

Comment:
🤣 Love it!

Only because I usually think about this at work, but should we be a bit more formal? 🤔

Reply:
Nope, it's a fact so they may as well know what I think about it.